src/components/MenuBar/MenuBar.style.ts

```diff
@@ -0,0 +1,117 @@
+import styled from 'styled-components';
+
+// Menu Bar container styling
+export const StyledMenuBar = styled.div`
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	background-color: #e97462; /* Primary Color */
```

Comment:
Please use the theme colours throughout. Syntax example:
`background-color: ${props => props.theme.colors.primary};`